en/option-gl/component/globe.md

```diff
@@ -51,14 +51,14 @@ BaseTexture: mapChart
 ```
 
 ## heightTexture(string|HTMLImageElement|HTMLCanvasElement)
-Height texture can be provided to render globe surface in three dimensions. Echarts uses [bump mapping] (https://zh.wikipedia.org/wiki/%E5%87%B9%E5%87%B8%E8%B4%B4%E5%9B%BE) which simulates bumps and wrinkles on the surface of an object. Below are two examples, one using heightTexture and another one without heightTexture:  
+Height texture can be provided to render globe surface in three dimensions. Echarts uses [bump mapping] (https://en.wikipedia.org/wiki/Bump_mapping) which simulates bumps and wrinkles on the surface of an object. Below are two examples, one using heightTexture and another one without heightTexture:  
```

Comment:
The space after `[bump mapping]` should be removed. The link of markdown is in the form of `[xxx](link)` rather than `[xxx] (link)`.